README.md

```diff
@@ -20,12 +20,17 @@ Planned features in no particular order:
 - More administration features
 - Update configs
 
+You might also find useful information in the [Online VScode Kafka Documentation](https://github.com/jlandersen/vscode-kafka/blob/master/docs/README.md)
+or you can read this documentation inside vscode with the command `Open VScode Kafka Documentation` available with `Ctrl+Shift+P`:
```

Comment:
inside vscode => inside your editor

can run in vscodium, che, theia...

Reply:
fixed